addons/popochiu/editor/importers/aseprite/docks/aseprite_importer_dock.gd

```diff
@@ -359,7 +363,10 @@ func _on_tags_title_toggled(button_pressed: bool) -> void:
 
 
 func _set_tags_visible(is_visible: bool) -> void:
-	get_node("%Tags").visible = is_visible
+	# If the tags container is empty, we show an info box
+	get_node("%TagsInfo").visible = (get_node("%Tags").get_child_count() == 0)
+
+	get_node("%TagsScrollContainer").visible = is_visible
```

Comment:
Are these uniquely named nodes? Because if they are, you could do:

```suggestion
	%TagsInfo.visible = %Tags.get_child_count() == 0

	%TagsScrollContainer.visible = is_visible
```

Otherwise, ignore this comment.